docs/static/dead-letter-queues.asciidoc

```diff
@@ -44,10 +44,10 @@ dead_letter_queue.enable: true
 
 Dead letter queues are stored as files in the local directory of the Logstash
 instance. By default, the dead letter queue files are stored in
-`path.data/dead_letter_queue`. Each pipeline has a separate queue. For example,
+`%{path.data}/dead_letter_queue`. Each pipeline has a separate queue. For example,
 the dead letter queue for the `main` pipeline is stored in
-`LOGSTASH_HOME/data/dead_letter_queue/main` by default. The queue files are
-numbered sequentially: `1.log`, `2.log`, and so on.
+`${LOGSTASH_HOME}/data/dead_letter_queue/main` by default. A queue is composed of
+one or more sequentially-numbered files (e.g., `1.log`, `2.log`, and so on).
```

Comment:
Please consider removing `e.g.,` It's an abbreviation of a latin phrase, and isn't considered translation friendly.  I use "for example" instead, but in this case, you might not even need that.